src/applications/hca/components/CustomReviewField.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const CustomReviewField = ({
+  children: {
+    props: { uiSchema, formData },
+  },
+}) => (
+  <div className="review-row">
```

Comment:
Needs to be a `dl>`

Reply:
I think the wrapper of this component is a `<dl>`. Axe doesn't throw any errors if there is a single `<div>` wrapping the `<dt>` & `<dd>` children.

Reply:
hey @timwright12 👋  ... thanks for the call out! So I looked into using the `<dl>` tag here but looks like the review page already provides one so were using a `div` to wrap the name value group for the `review-row` styling here ... https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#wrapping_name-value_groups_in_div_elements verified the elements pass accessibility checks as well


![Apply for Health Care  Veterans Affairs 2021-11-09 09-21-56](https://user-images.githubusercontent.com/7518899/140941667-7cd03fe6-6879-4500-af97-9d890c4767ca.png)

Reply:
@timwright12 @Mottie thanks for the informative review, always learn something new